packages/splits-sdk/src/client/base.ts

```diff
@@ -470,6 +153,7 @@ export class BaseTransactions extends BaseClient {
         functionName,
         account: this._walletClient.account,
         args: functionArgs ?? [],
+        value,
```

Comment:
nit: do you want to add `value` into the calldata response as well? i know it's not really needed, doesn't impact the actual `calldata`. maybe for completeness it's nice to have it there?